cmds/vboot/vboot.go

```diff
@@ -0,0 +1,122 @@
+package main
```

Comment:
Please add a package comment explaining this tool, and briefly explain its usage.